### DISCUSSION TITLE
Octolamp build by Jeroen

### DISCUSSION THREAD

**Discussion by jeroenniesen** [martinwoodward/octolamp#2] [Show and tell]
As a big fan of GitHub I couldn't resist to print this lamp and solder the lightstrip and module inside of the lamp.

I printed the lamp using a Creality Ender S1 Pro (connected using the Creality Sonic Pad)

I used some sticky tape that sticks on both sides to stick the module to the back side of the lamp. Leaving it loose resulted in shadows that were visible.

Here are some pictures of my build:
![image](https://user-images.githubusercontent.com/24291535/213309676-5842efb3-27a5-4fef-807c-48335c3578c4.jpeg)
![image](https://user-images.githubusercontent.com/24291535/213313655-649887cc-025f-428c-8bee-742432734a49.jpeg)![image](https://user-images.githubusercontent.com/24291535/213313695-e7c4d709-8029-4599-93fe-027631b185ca.jpeg)
https://user-images.githubusercontent.com/24291535/213309715-b8c1c0dd-78d0-4b3b-aac0-76491a84ff44.MOV

**Comment by martinwoodward** [comment]
Looks amazing!! The controller was supposed to fit in that void between the reflector and the black base - did that not work for you?

**Reply by jeroenniesen** [reply]
@martinwoodward I used the connectors that came with the board (and were quite big). So it didn't fit in that space anymore.

I already came to the conclusion that I have to desolder the connector and make it fit behind the ledstrip area. The board has a bright led on it which is visible now... 

Will be nice work for the weekend 😊

**Reply by martinwoodward** [reply]
Ah yeah, I was wondering if I should make the void deeper so that JST connectors fit better. I reckon it would only mean making the base base 2 or 3mm deeper.

Love to know what people think. The goal with this build was to make something easy, approachable and affordable.

**Reply by martinwoodward** [reply]
Actually, read that wrong and it was header pins rather than the JST plugs which take even more room.

Perhaps I should do a variant that is deeper in the base?  That would allow for a bunch of the esp8266 boards that come with pre-soldered headers as well..

Love to know if people would find that useful

**Reply by barkby** [reply]
Problem you have is there are so many versions of the ESP boards. I used an ESP32 C3 initially for mine and could not flash it as WED.ME so had to go back to an older version of ESP board.
ESP32 C3 are not supported with WLED right now (plus the ESP32 S2 is not and probably wont be supported either)

**Reply by metcalfc** [reply]
> Love to know if people would find that useful

This would be great. I used right angle headers to minimize the height but the plug was the challenge to put it behind the reflector.
